src/brokers/broker-details/components/BrokerDetailsBreadcrumb/BrokerDetailsBreadcrumb.test.tsx

```diff
@@ -35,10 +62,41 @@ describe('BrokerDetailsBreadcrumb', () => {
     expect(comp.getByText(`${podName}`)).toBeInTheDocument();
   });
 
+  it('should navigate back to BrokerList page when click on Brokers BreadcrumbItem', async () => {
+    const comp = render(
+      <MemoryRouter>
+        <BrokerDetailsBreadcrumb
+          name={name}
+          namespace={namespace}
+          podName={podName}
+        />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByText('brokers'));
+    expect(navigate).toHaveBeenCalledWith(`/k8s/ns/${namespace}/brokers`);
+  });
+
+  it('should navigate back to BrokerPodsList page when click on BrokerPods BreadcrumbItem', async () => {
+    const comp = render(
+      <MemoryRouter>
+        <BrokerDetailsBreadcrumb
+          name={name}
+          namespace={namespace}
+          podName={podName}
+        />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByText(`broker ${name}`));
+    expect(navigate).toHaveBeenCalledWith(
+      `/k8s/ns/${namespace}/brokers/${name}`,
+    );
+  });
+
```

Comment:
It feels like these tests aren't related to the PR, can you postpone them to another one please?